gdnative/src/lib.rs

```diff
@@ -53,12 +53,12 @@
 
```

Comment:
The goal of this change is to remove the workaround that was previously put in place. To confirm, the reason for this change is that the workaround is no longer needed, correct? If so, I think removing all the references as well as the comment may be the best option.

Reply:
Definitely, this is still a WIP and I need to first confirm that the workaround is no longer needed (at least on stable Rust).

Reply:
I built the docs from your branch and currently it looks like regardless of what we do we're still getting the multiple module paths when searching the docs.

Your changes: ![image](https://user-images.githubusercontent.com/1459514/135967701-902805e4-c3c7-4151-afb5-b28bd4bc5cdc.png)
https://godot-rust.github.io/docs/gdnative/ ![image](https://user-images.githubusercontent.com/1459514/135967897-3a9caea1-e8f6-4bfa-a24e-d631392e784c.png)